{
  "_from": "rn-test-exception-handler@^1.3.0",
  "_id": "rn-test-exception-handler@1.3.1",
  "_inBundle": false,
  "_integrity": "sha512-p5sTn/pOIPumXTjAEjUESC6gSBSqsN5QazeEOKhheE3yaAsCIb9i668OWjzn2L/ap9sKIawnX3/cHl8xzIhlFg==",
  "_location": "/rn-test-exception-handler",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "rn-test-exception-handler@^1.3.0",
    "name": "rn-test-exception-handler",
    "escapedName": "rn-test-exception-handler",
    "rawSpec": "^1.3.0",
    "saveSpec": null,
    "fetchSpec": "^1.3.0"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/rn-test-exception-handler/-/rn-test-exception-handler-1.3.1.tgz",
  "_shasum": "90146bc04e02f63d2a675675d2ebae5a2b8e9a06",
  "_spec": "rn-test-exception-handler@^1.3.0",
  "_where": "/home/user/Desktop/react-native-exception-handler-example",
  "author": "",
  "bundleDependencies": false,
  "deprecated": false,
  "description": "A react native module to purposefully raise a Native Exception to try and crash the app.",
  "keywords": [
    "react-native"
  ],
  "license": "",
  "main": "index.js",
  "name": "rn-test-exception-handler",
  "peerDependencies": {
    "react-native": "*"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "version": "1.3.1"
}
